model/common/src/icon4py/model/common/grid/base.py

```diff
@@ -183,6 +183,14 @@ def _construct_connectivity(self, dim, from_dim, to_dim):
             )
         else:
             neighbor_table = self._neighbor_tables[dim]
+
+        if (dimension_size := self.size[to_dim]) <= neighbor_table.shape[0]:
```

Comment:
please spend a minute in verifying that `to_dim` is correct. (but I guess tests would catch it anyway)

Reply:
Is it not the other way round? for C2E: `neighbor_table.shape[0] == num_cells` and we use:
```
 "C2E": (self._construct_connectivity, dims.C2EDim, dims.CellDim, dims.EdgeDim),
``` 
so `CellDim == from_dim`
The naming reflects the mapping of the indices not the fields.

Reply:
tests seem to say to support that you need to use `from_dim`
```suggestion
        if (dimension_size := self.size[from_dim]) <= neighbor_table.shape[0]:
```
Other thought, could it cause problems with the 1d connectivities?